x/auth/keeper/account.go

```diff
@@ -28,7 +28,7 @@ func (ak AccountKeeper) NewAccount(ctx sdk.Context, acc types.AccountI) types.Ac
 // GetAccount implements AccountKeeperI.
 func (ak AccountKeeper) GetAccount(ctx sdk.Context, addr sdk.AccAddress) types.AccountI {
 	store := ctx.KVStore(ak.key)
-	bz := store.Get(types.AddressStoreKey(addr))
+	bz := store.Get(types.AddressStoreKey("", addr)) //TODO: figure out how to pass in the type
```

Comment:
not sure of how to proceed with this without making a breaking change. @alexanderbez suggested using bech32 HRP but the prefix is the same for different account types (base, vesting, module, etc). Any ideas? @amaurymartiny @aaronc @AdityaSripal @clevinson ?